adafruit_gps.py

```diff
@@ -91,49 +99,49 @@ def _parse_degrees(nmea_data):
     return degrees + minutes  # return parsed string in the format dddmmmmmm
```

Comment:
I ended up just removing it since the comments below the function definition say it's "a pure degrees value".  I think that's clear between that, the other comments, and the type annotation, but happy to change to improve clarity.  Good catch!